docs: Clarify and update docs (#1024)
thousandbrainsproject/tbp.monty: docs/contributing/contributing.md

title: Ways to Contribute
---

> Welcome, and thank you for your interest in contributing to Monty! (If you need more reasons why getting involved in this project might be a great way to dedicate your time, see our [why contribute?](./why-contribute.md) page)

Reviewer: ```suggestion
Welcome, and thank you for your interest in contributing to Monty! (If you need more reasons for why getting involved in this project might be a great way to dedicate your time, see our [why contribute?](./why-contribute.md) page)
```

Monty integrates code changes using GitHub Pull Requests. For details on how Monty uses Pull Requests, please consult the [Contributing Pull Requests](ways-to-contribute-to-code.md) guide.

# Test Our Approach/Build a Demo

> The vision of this project is to implement a generally intelligent system that can solve a wide variety of sensorimotor tasks (at a minimum, any task the neocortex can solve/humans can perform with ease). To evaluate this, we are continually looking for test beds in which we can assess the system's capabilities. If you have an idea of how to test an important capability or you have an existing benchmark on which you want to compare our algorithm, please consider contributing these. You can have a look at our [Project Showcase](../community/project-showcase.md) page for some inspiration from previous Monty demos.

Reviewer: ```suggestion
The vision of this project is to implement a generally intelligent system that can solve a wide variety of sensorimotor tasks (at a minimum, any task the neocortex can solve that humans can perform with ease). To evaluate this, we are continually seeking ways to assess the system's capabilities. If you have an idea for how to test an important capability or an existing benchmark to compare our algorithm against, please consider contributing these. You can have a look at our [Project Showcase](../community/project-showcase.md) page for some inspiration from previous Monty demos.
```
